### PR TITLE
Abodzas/avsw pdr

### PR DESCRIPTION
**Pull request by Beaudidly**
It still needs a bit of an overview section :/

### CODE REVIEW

File: docs/pdr.md [hunk 1]
```diff
@@ -0,0 +1,247 @@
...
+* Report low battery flag when low on battery based off coulomb count to the MFC**
+* Report voltages of 5V and 12V rails to the MFC
+* Report currents of 5V and 12V rails to the MFC
+* Report board [thermistor] temperature reading to MFC
+
+#### Environment
+**Bare Metal**:
+The tasks performed by this card are simple. A simple while loop blocking on a periodic clock should suffice.  Some hardware configuration needs to occur during an initialization period.
+
+## Verification Plans
```
Comment (philiplinden) [inline docs/pdr.md]:
```suggestion
### Verification Plans
```

File: docs/pdr.md [hunk 2]
```diff
@@ -0,0 +1,247 @@
+# PDR
```
Comment (philiplinden) [inline docs/pdr.md]:
```suggestion
# Preliminary Design
```
This page showcases our preliminary designs, but not how they were reviewed. I think we should call the page what it is. I will add an explanation about what is captured here in the "overview" section.

File: docs/pdr.md [hunk 3]
```diff
@@ -0,0 +1,247 @@
...
+
+#### Board Level
+Test each board at it's interface.  For MFC this would involve stimulating the ETH/CAN interfaces, and observing the output over the same interfaces.  The other MCU boards are trickier.  Their CAN bus can be easily stimulated/inspected, but behavior of the actuators/sensors is tricker to control and inspect.  More test harnessing would be involved, but perhaps already accomplished to meet system level testing.
+
+#### MFC App Level Testing
+The MFC is running a full operating system, which in turn executes discrete applications.  These applications can be tested individually for proper behavior.  A Software in the Loop test environment can be created to allow for focused inspected on each of these apps.  Unit Testing frameworks can also be used to verify behavior.
+
+## Risk Assessment
```
Comment (philiplinden) [inline docs/pdr.md]:
```suggestion
### Risk Assessment
```

File: docs/pdr.md [hunk 4]
```diff
@@ -0,0 +1,247 @@
...
+### Hazard Identification, Analysis, Mitigation
```
Comment (philiplinden) [inline docs/pdr.md]:
```suggestion
#### Hazard Identification, Analysis, Mitigation
```

File: docs/pdr.md [hunk 5]
```diff
@@ -0,0 +1,247 @@
...
+
+**Actuator Card Failure** (Prob=1, Sev=2):  The cards driving the ballast or balloon valve actuators might fail in an undesired state.  This may cause for unsafe conditions by fully emptying the ballast or balloon at an inappropriate time.  The mission might uncontrollably be extended or shortened. Watchdogs could be used to attempt to restart these boards, but are not guaranteed to work if the hardware is permanently damaged.
+
+**MFC Real Time Failure** (Prob=1 Sev=1):  The MFC slips on meeting it’s required timely behavior.  This may be slipping on transmitting a message on time, or not reacting to a control change fast enough.  A lack of real time behavior in these areas will cause for a lack of control performance.  But given the MFC is using an application processor that far passes our needs, this is not a likely outcome, and not a frequent one if that.
+
+## Cost
```
Comment (philiplinden) [inline docs/pdr.md]:
```suggestion
### Cost
```

File: docs/pdr.md [hunk 6]
```diff
@@ -0,0 +1,247 @@
...
+$0, expecting to strictly use “free as in beer” software.
+
+But for developing pre-hardware release, devboards will be used.  $65 for the Beaglebone Black and $25 for a STM32F4 dev board for a total of **$90**.
+## Schedule
```
Comment (philiplinden) [inline docs/pdr.md]:
```suggestion
### Schedule
```

File: mkdocs.yml [hunk 7]
```diff
@@ -4,6 +4,7 @@ nav:
   - Mission Concepts: concepts.md
   - Concept Selection: concept-selection.md
   - Mission Profile: mission-profile.md
+  - PDR: pdr.md
```
Comment (philiplinden) [inline mkdocs.yml]:
```suggestion
  - Preliminary Design: pdr.md
```

File: docs/pdr.md [hunk 8]
```diff
@@ -0,0 +1,247 @@
+# PDR
+**Overview**:
```
Comment (philiplinden) [inline docs/pdr.md]: Maybe convert this to a comment for now.

**Review by philiplinden — CHANGES_REQUESTED** [review]
I'll have to see how this format works out for multiple subsystems, but for avsw alone it works pretty well. 
I made a bunch of nitpick suggestions to formatting. The heading level is pretty critical though as it dictates the sidebar tree.

**Review by philiplinden — APPROVED** [review]

File: docs/pdr.md (hunk 8 above)
Comment (Beaudidly) [reply docs/pdr.md]: This should be fixed!

**Pull request merged**